components/movementsensor/gpsrtk/gpsrtk.go

```diff
@@ -61,9 +59,11 @@ type gpsrtk struct {
 }
 
 func (g *gpsrtk) start() error {
-	g.activeBackgroundWorkers.Add(1)
-	utils.PanicCapturingGo(g.receiveAndWriteCorrectionData)
-	return g.err.Get()
+	if g.workers != nil {
+		return errors.New("do not double-start background goroutines")
```

Comment:
Can we simplify this error message more? I have no idea what this means

Reply:
Sure! The goal is that you should never call `start()` twice on the same object, because that will leak goroutines. At the moment, this never happens (`start()` is only ever called from a constructor), but if that ever changes in the future, it'll be a problem (recall the trouble we discovered several months ago when the gpsnmea component started within a gpsrtk component started its own goroutine and then the gpsrtk component that contained it started a second copy). 

Any suggestions what the error message should say instead?

Reply:
Maybe say that! `"do not call start() twice on the same object"`